src/render_tree.rs

```diff
@@ -215,148 +177,61 @@ impl Node {
             position: Position::new_from_existing(position),
         };
 
-        position.offset_y(font_size);
-        position.offset_y(margin);
+        position.offset_y(fs + margin);
 
         new_node
     }
 
-    pub fn new_heading2(position: &mut Position) -> Self {
-        let margin = 9.96;
-        position.offset_y(margin);
+    // I took the margins/font sizes from Chrome dev tools.
+    // There are still some slight differences but it's very close
 
-        let new_heading = TextNode::new_heading2();
-        let font_size = new_heading.font_size;
+    pub fn new_heading1(position: &mut Position) -> Self {
+        let margin = 10.72;
```

Comment:
It might make sense to remove the variable for the margin and just write it directly in `Node::new_text`

Reply:
yes, I'd probably agree with that.